packages/manager/modules/pci/src/projects/project/edit/edit.controller.js

```diff
@@ -0,0 +1,99 @@
+import pick from 'lodash/pick';
+
+import { DEFAULT_PROJECT_KEY, MESSAGES_CONTAINER_NAME } from './edit.constant';
+
+export default class ProjectEditController {
+  /* @ngInject */
+  constructor($stateParams, $translate, CucCloudMessage, ovhUserPref, OvhApiCloudProject) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.ovhUserPref = ovhUserPref;
+    this.OvhApiCloudProject = OvhApiCloudProject;
+
+    this.loading = {
+      init: false,
+      submit: false,
+    };
+
+    this.$ngInit();
+  }
+
+  $ngInit() {
```

Comment:
Current convention is to use `$onInit` hook instead. We should probably talk if there is a difference and what's better especially with performances ( FYI : https://docs.angularjs.org/api/ng/service/$compile#life-cycle-hooks only mentions `$onInit`)

Reply:
Will switch to component instead of controller / view